<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="5" failed="0" ignored="0" total="5" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
    </line>
    <line>
      <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
    </line>
    <line>
      <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
    </line>
    <line>
      <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
    </line>
    <line>
      <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
    </line>
    <line>
      <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
    </line>
    <line>
      <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
    </line>
    <line>
      <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
    </line>
    <line>
      <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
    </line>
    <line>
      <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
    </line>
  </reporter-output>
  <suite name="Login Test Cases" duration-ms="41635" started-at="2021-08-25T10:46:27Z" finished-at="2021-08-25T10:47:09Z">
    <groups>
    </groups>
    <test name="TC-1. Blank UserName validation." duration-ms="41635" started-at="2021-08-25T10:46:27Z" finished-at="2021-08-25T10:47:09Z">
      <class name="com.index.AddCartIndex">
        <test-method status="PASS" signature="fetchSuiteConfiguration(org.testng.ITestContext)[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="fetchSuiteConfiguration" is-config="true" duration-ms="271" started-at="2021-08-25T16:16:27Z" finished-at="2021-08-25T16:16:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@6f3c660a]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fetchSuiteConfiguration -->
        <test-method status="PASS" signature="one(org.testng.ITestContext)[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="one" is-config="true" duration-ms="1" started-at="2021-08-25T16:16:27Z" finished-at="2021-08-25T16:16:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@6f3c660a]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- one -->
        <test-method status="FAIL" signature="init()[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="init" is-config="true" duration-ms="0" started-at="2021-08-25T16:16:27Z" finished-at="2021-08-25T16:16:27Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.index.LoginIndex.init(LoginIndex.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="setUp" is-config="true" duration-ms="2577" started-at="2021-08-25T16:16:28Z" finished-at="2021-08-25T16:16:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.index.AddCartIndex.addProductToCart() throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@6f3c660a]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
            </line>
            <line>
              <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="addProductToCart()[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="addProductToCart" duration-ms="0" started-at="2021-08-25T16:16:30Z" finished-at="2021-08-25T16:16:30Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.index.LoginIndex.init(LoginIndex.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addProductToCart -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="setUp" is-config="true" duration-ms="3495" started-at="2021-08-25T16:16:30Z" finished-at="2021-08-25T16:16:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.index.LoginIndex.login() throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@6f3c660a]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
            </line>
            <line>
              <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="login()[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="login" duration-ms="0" started-at="2021-08-25T16:16:34Z" finished-at="2021-08-25T16:16:34Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.index.LoginIndex.init(LoginIndex.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="setUp" is-config="true" duration-ms="4133" started-at="2021-08-25T16:16:34Z" finished-at="2021-08-25T16:16:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.index.LoginIndex.login_blankPassword() throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@6f3c660a]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
            </line>
            <line>
              <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="login_blankPassword()[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="login_blankPassword" duration-ms="0" started-at="2021-08-25T16:16:38Z" finished-at="2021-08-25T16:16:38Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.index.LoginIndex.init(LoginIndex.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_blankPassword -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="setUp" is-config="true" duration-ms="3735" started-at="2021-08-25T16:16:38Z" finished-at="2021-08-25T16:16:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.index.LoginIndex.login_blankUserName() throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@6f3c660a]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
            </line>
            <line>
              <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="login_blankUserName()[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="login_blankUserName" duration-ms="0" started-at="2021-08-25T16:16:41Z" finished-at="2021-08-25T16:16:41Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.index.LoginIndex.init(LoginIndex.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_blankUserName -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="setUp" is-config="true" duration-ms="27629" started-at="2021-08-25T16:16:41Z" finished-at="2021-08-25T16:17:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.index.LoginIndex.login_negative() throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@6f3c660a]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[<br>current test- TC-1. Blank UserName validation.]]>
            </line>
            <line>
              <![CDATA[Current Xml Suite is:---->Login Test Cases]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="login_negative()[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="login_negative" duration-ms="0" started-at="2021-08-25T16:17:09Z" finished-at="2021-08-25T16:17:09Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.index.LoginIndex.init(LoginIndex.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_negative -->
        <test-method status="SKIP" signature="closebrowser()[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="closebrowser" is-config="true" duration-ms="0" started-at="2021-08-25T16:17:09Z" finished-at="2021-08-25T16:17:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closebrowser -->
        <test-method status="SKIP" signature="testResults()[pri:0, instance:com.index.AddCartIndex@1e397ed7]" name="testResults" is-config="true" duration-ms="0" started-at="2021-08-25T16:17:09Z" finished-at="2021-08-25T16:17:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testResults -->
      </class> <!-- com.index.AddCartIndex -->
    </test> <!-- TC-1. Blank UserName validation. -->
  </suite> <!-- Login Test Cases -->
</testng-results>
